pom.xml

```diff
@@ -45,7 +45,7 @@
         <ojdbc6.version>11.2.0.4</ojdbc6.version>
         <opencsv.version>4.1</opencsv.version>
         <plexus-utils.version>3.0.24</plexus-utils.version>
-        <mockito.version>3.11.2</mockito.version>
+        <mockito.version>4.0.0</mockito.version>
         <powermock.version>2.0.7</powermock.version>
         <!--Cornell Customization: backport redis-->
         <spring-data-redis.version>2.6.0</spring-data-redis.version>
```

Comment:
The 2021-11-17 financials patch appears to be using older versions of the Lettuce and Spring Data Redis dependencies than what we use. You may need to add appropriate KFS-dependency-based and WAR-filtering-based exclusions so that we don't accidentally use the older versions of those.